app/webpacker/css/admin/products_v3.scss

```diff
@@ -108,8 +107,12 @@
   }
 
   #filters {
-    gap: 20px;
-    align-items: flex-end;
+    display: grid;
+    grid-template-columns: repeat(6, 1fr);
+    grid-template-rows: 1fr;
+    grid-column-gap: 20px;
```

Comment:
> which leads me to ask if there's a framework we can use?

Why do you want to use a framework here? To be more easy to use? I can think of [tailwind](https://tailwindcss.com/docs/grid-template-columns) for sure.

> I've used 12-column grids before and found them fine, what do you think? A 12-column grid could work here.

Could you please let me know where did you use this? I can then remplace my code by yours.

Reply:
In another project (members.ceresfairfood.org.au) we use Foundation's XY Grid (https://get.foundation/sites/docs/xy-grid.html) but I wouldn't necessarily consider it the best. Tailwind looks quite good, I like how you can specify the grid columns easily. I guess this case would look something like:

<div class="grid grid-cols-6 gap-4">
  <div class="col-span-3">query</div>
  <div>etc</div>
</div>

I know it sort of breaks rules, putting styling rules within the markup. 
And you could argue the native CSS code is not that hard to read (once you're used to it).
I would expect adding something fully featured like Tailwind would increase our CSS size significantly, which is something to consider.

It certainly is a question of opinion.. I guess my main concern is to not make up rules if there are existing rules we can follow. 
Anyway, I think I'm overthinking it now, just wondering what your opinion was.

Reply:
> I would expect adding something fully featured like Tailwind would increase our CSS size significantly, which is something to consider.

We try to use tailwind previously (that was one of my idea I wanted to push), but without success actually because it was difficult to integrate tailwind to an existing (and complex) project. But I'm in favor of using framework for sure!